README.md

```diff
@@ -104,6 +115,11 @@ You can and should overwrite these during launching your container:
     bgruening/galaxy-stable
   ```
 
+Note that if you would like to run any of the [cleanup scripts](https://wiki.galaxyproject.org/Admin/Config/Performance/Purge%20Histories%20and%20Datasets), you will need to add the following to `/export/galaxy-central/config/galaxy.ini`:
+
```

Comment:
This is ok, just wanted to let you know that the "correct" approach would be to improve the cleanup scripts to use Galaxy build-in config parser. This will also read ENV vars as well and would make this step unnecessary.